tools/update_cross_build_releases.py

```diff
@@ -15,6 +19,10 @@
 
 BASE_URL = "https://github.com/pyodide/pyodide/releases/download/{version}/xbuildenv-{version}.tar.bz2"
 
+# Pyodide build version that is compatible with the latest cross-build environment
+# Note for maintainers: update this value when there is a breaking changes in the cross-build environment
+MIN_COMPATIBLE_PYODIDE_BUILD_VERSION = "0.26.0"
```

Comment:
Could we move this to `Makefile.envs` in a followup? I don't think people will be looking at this file very much.

Reply:
Sure, my goal is not to change that value as much as possible, but if that happens, let's move it to the Makefile.